pkg/controller/activemqartemis/activemqartemis_controller_test.go

```diff
@@ -0,0 +1,202 @@
+package activemqartemis
+
+import (
+	"fmt"
+	routev1 "github.com/openshift/api/route/v1"
+	brokerv2alpha1 "github.com/rh-messaging/activemq-artemis-operator/pkg/apis/broker/v2alpha1"
+	"github.com/rh-messaging/activemq-artemis-operator/pkg/resources/services"
+	"github.com/rh-messaging/activemq-artemis-operator/pkg/resources/statefulsets"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"strconv"
+	"testing"
+)
+
+func TestReconcilerImplementsInterface(t *testing.T) {
+	logf.SetLogger(logf.ZapLogger(true))
+	reconciler := ReconcileActiveMQArtemis{}
+	var i interface{} = &reconciler
```

Comment:
hmm, not sure why we'd test this... that it implements generic Go interface{}? Everything does!

Reply:
Removed, Not required, for this operator, I have added because I was referring the OCS operator.